AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md

```diff
@@ -46,9 +53,9 @@ In the end, you will have two new conda environments which are user-tensorflow-g
 ## How to Build and Run
 
 You can run the Jupyter notebook with the sample code on your local
-server or download the sample code from the notebook as a Python file and run it locally or on the Intel DevCloud.
+server.
+
 
-**Note:** You can run this sample on the Intel DevCloud using the Dask and OmniSci engine backends for Modin. To learn how to set the engine backend for Intel Distribution of Modin, visit the [Intel® Distribution of Modin Getting Started Guide](https://www.intel.com/content/www/us/en/developer/articles/technical/intel-distribution-of-modin-getting-started-guide.html). The Ray backend cannot be used on Intel DevCloud at this time. Thank you for your patience.
 
 ### Run the Sample in Jupyter Notebook<a name="run-as-jupyter-notebook"></a>
 
```

Comment:
At the end of the file we need to insert:
*Other names and brands may be claimed as the property of others. [Trademarks](http://www.intel.com/content/www/us/en/legal/trademarks.html)

This request is relevant for any README file where we use asterisks

Reply:
addressed it accordingly